### ISSUE TITLE
Debian 11 Client Installation Problem

### ISSUE THREAD

**Issue by ldfloyd** [Libki/libki-client#82]
I ran the amended command and then built the project (with no errors) in Qt Creator. Moving onto stage 5.2.2., I'm now presented with the attached error.
![5 2 2  Section](https://user-images.githubusercontent.com/66160100/173683366-31f67fbe-760a-42a3-b2ea-6b017d60d6af.JPG)

**Comment by ldfloyd** [comment]
![dir1](https://user-images.githubusercontent.com/66160100/173687563-8d4f9364-6f75-4694-8380-fa97e66ba213.JPG)
![drivesearch](https://user-images.githubusercontent.com/66160100/173687634-ef29a4a0-0987-42db-90e6-473cf6ece5d1.JPG)

I'm not sure what I'm looking for exactly.

**Comment by loidor** [comment]
Being in Sweden, I'm just about to go to bed, but I'll record a video at
work tomorrow on how to do it and post the youtube link once it's up (I
guess about 15 hours from now).

**Comment by ldfloyd** [comment]
Cheers - I'm in the UK, so off to bed too!

**Comment by loidor** [comment]
All right, here we go! I'll probably make a proper video with actual scripting later. I realised I made a couple of redundant ```cd```commands, which is basically to show where the files end up in relation to each other. I.e. you get one directory with the libki-client files and next to that one you get the build directory which contains the actual client. The last command is just to show that it runs. 

You will want to run it through autorun on the actual machines though, so open the menu and find Startup (at least I _think_ it's Startup in Debian as well as on Ubuntu) and add a custom startup. Set it to one second or something like that and enter the custom command ```libkiclient```.
[](https://www.youtube.com/watch?v=U0X2hs0IPI8)
